monkey/agent_plugins/exploiters/hadoop/config-schema.json

```diff
@@ -0,0 +1,43 @@
+{
+    "title": "Hadoop/Yarn Exploiter",
+    "description": "Remote code execution on HADOOP server with YARN and default settings. Logic based on https://github.com/vulhub/vulhub/tree/master/hadoop/unauthorized-yarn.",
+    "required": [
+        "exploitable_ports",
+        "http_download_timeout",
+        "app_random_string_length"
+    ],
+    "properties": {
+        "exploitable_ports": {
+            "title": "List of exploitable ports",
+            "description": "A list of potential exploitable ports that Hadoop will try to exploit.",
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 65535
+            },
+            "default": [8088]
+        },
+        "http_download_timeout": {
+            "title": "HTTP server download timeout",
+            "description": "How long the http server will be open for downloads in seconds.",
+            "type": "number",
+            "minimum": 0,
+            "default": 60
+        },
+        "app_random_string_length": {
+            "title": "Length of random string",
+            "description": "Random string length that is used for creating a unique application name.",
+            "type": "number",
+            "minimum": 5,
+            "default": 6
+        },
+        "id_string": {
+            "title": "Identification string for hadoop app name",
+            "description": "Identification string which will be appended to the unique application name.",
+            "type": "string",
+            "default": "M0NK3Y3XPL0ITABLE"
+        }
+    }
+}
```

Comment:
What about
1. the path where the Agent binary is dropped?
2. the timeout used in `requests.post()` calls?

Reply:
1. @mssalvatore  ?
2. Added `request_timeout`.

Reply:
Yep!

Reply:
After some discussion, we all agree this would be nice, but it's more complicated than it's worth today. We may add it in v2 of this exploiter.